src/Controller/PurposeValuesController.php

```diff
@@ -29,32 +30,33 @@ public function newPurposeValues(
         string $type
     ): Response {
         $home = new Home();
+        $home->setType($type);
+        $home->setTitle("null");//because there is no title
+        $home->setPictureTwo("null");//because there is no picture2
+        $home->setPictureThree("null");//because there is no picture3
+        $home->setLegendPictureTwo("null");//because there is no picture2
+        $home->setLegendPictureThree("null");//because there is no picture3
```

Comment:
because there is no LEGEND picture ?

Reply:
There is only one picture in purpose article and values article so there is lo legend